armometadata/k8sutils.go

```diff
@@ -106,7 +106,7 @@ func LoadConfig(configPath string) (*ClusterConfig, error) {
 }
 
 // ExtractMetadataFromBytes extracts metadata from the JSON bytes of a Kubernetes object
-func ExtractMetadataFromJsonBytes(input []byte) (error, map[string]string, map[string]string, map[string]string, string, string, string, string) {
+func ExtractMetadataFromJsonBytes(input []byte) (error, map[string]string, map[string]string, map[string]string, string, string, string, string, map[string]string) {
```

Comment:
at some point we should consider returning a struct to avoid breaking api all the time

Reply:
see this commit:
https://github.com/armosec/utils-k8s-go/pull/13/commits/d145476d10e0b08d471f36bddaa660230f7fbec5

@matthyx